bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "remotestorage-modules",
+  "description": "remotestorage modules collection",
+  "keywords": "remotestorage modules collection",
+  "main": "src/*.js",
+  "ignore": [
+  ],
+  "version": "0.0.1",
+  "private": false,
+  "author": "Niklas Cathor <nilclass@riseup.net>",
```

Comment:
Unfortunately, @nilclass has given up on all public open-source work some time ago, and these modules are written by a whole bunch of people. I think the key supports an array, right (or maybe that was just npm)? Maybe we should just list all names from the contributors page and do the same for remotestorage.js.